iroha/benches/validation.rs

```diff
@@ -141,10 +122,10 @@ fn validate_transaction(criterion: &mut Criterion) {
     .expect("Failed to accept transaction.");
     let mut success_count = 0;
     let mut failures_count = 0;
-    let mut world_state_view = WorldStateView::new(Peer::new(PeerId::new(
-        "127.0.0.1:8080",
-        &key_pair.public_key,
-    )));
+    let mut world_state_view = WorldStateView::new(
+        Peer::new(PeerId::new("127.0.0.1:8080", &key_pair.public_key)),
```

Comment:
I don't understand why peer is here.

Reply:
As we discussed in the chat, I removed Peer now from WSV, as it is no longer needed there.